cloudbuild.yaml

```diff
@@ -164,21 +164,21 @@ steps:
 
   - name: 'gcr.io/oak-ci/oak:latest'
     id: check_formatting
-    waitFor: ['git_check_diff', 'build_android_image']
+    waitFor: ['git_init', 'build_android_image']
     timeout: 30m
     entrypoint: 'bash'
     args: ['./scripts/check_formatting']
 
   - name: 'gcr.io/oak-ci/oak:latest'
     id: check_cargo_deny
-    waitFor: ['check_formatting']
+    waitFor: ['check_formatting', 'git_check_diff']
     timeout: 30m
     entrypoint: 'bash'
     args: ['./scripts/check_cargo_deny']
 
   - name: 'gcr.io/oak-ci/oak:latest'
     id: check_docs
-    waitFor: ['check_formatting']
+    waitFor: ['check_formatting', 'git_check_diff']
```

Comment:
I think this step and `check_cargo_deny` are small, inexpensive steps that can be done earlier without having to wait for `git_check_diff`. And IIUC they do not produce any files that need to be checked in the `git_check_diff` step. On the other hand, these steps do not fail as often as `check_formatting `, so it might not be as annoying to run them at the end...